### PR TITLE
Fix success action status

### PR DESCRIPTION
**Pull request by qasimzee**
My attempt to fix https://github.com/localstack/localstack/issues/1302
I agree to the contributor license agreement

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+asdfasd
```
Comment (whummer) [inline test.txt]: Can we please remove this file

File: tests/unit/test_s3.py [hunk 2]
```diff
@@ -30,17 +30,17 @@ def test_find_multipart_redirect_url(self):
                  b'redirect"\r\n\r\nhttp://127.0.0.1:5000/?id=20170826T181315.679087009Z\r\n--------------------------'
```
Comment (whummer) [inline tests/unit/test_s3.py]: Can we extend this test to include an example of `success_action_status`?

**Comment by whummer** [comment]
Thanks for this PR @qasimzee ! Looks great, just added two minor comments. Can we please address them before merging? Thanks

**Comment by qasimzee** [comment]
@whummer ping :)

File: localstack/services/s3/s3_listener.py [hunk 3]
```diff
@@ -686,17 +703,25 @@ def return_response(self, method, path, data, headers, response):
...
+            key, redirect_url = multipart_content.find_multipart_key_value(data, headers)
 
             if key and redirect_url:
                 response.status_code = 303
                 response.headers['Location'] = expand_redirect_url(redirect_url, key, bucket_name)
                 LOGGER.debug('S3 POST {} to {}'.format(response.status_code, response.headers['Location']))
 
+            key, status_code = multipart_content.find_multipart_key_value(
+                data, headers, 'success_action_status')
+            if response.status_code == 200 and status_code == '201' and key:
```
Comment (whummer) [inline localstack/services/s3/s3_listener.py]: One more thing - Seems that `success_action_status` could also contain status codes `200` or `204` (see https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectPOST.html). Shouldn't hold up the merge, but maybe we can add that as a separate PR @qasimzee ? That would be great, thanks.

**Comment by whummer** [comment]
Great, thanks for the quick turnaround @qasimzee ! Left another comment - would be great if we could tackle that in a small follow-up PR. Thanks

File: localstack/services/s3/s3_listener.py (hunk 3 above)
Comment (qasimzee) [reply localstack/services/s3/s3_listener.py]: let me put something this weekend

**Pull request merged**